### PR TITLE
[RFC][IMP][12.0] fieldservice_account

### PR DESCRIPTION
**Pull request by brian10048**
This PR adds the ability to create an invoice from a field service order with a fixed price and product, while still allowing a field service order to be invoiced based on the existing implementation using timesheets and/or contractor costs

### CODE REVIEW

**Comment by max3903** [comment]
@brian10048 What do you think of
- [x] adding a many2one field "Customer Invoice" to account.invoice on fsm.order
- [x] adding a many2one field "Vendor Bill" to account.invoice on fsm.order
- [x] hiding those fields if they are not set
- [x] setting those fields when the bill or invoice are created?

**Comment by brian10048** [comment]
@max3903 I can handle those items.

If there is anything else you would like changed, just let me know and I'll make the revisions.

**Comment by max3903** [comment]
@brian10048 Thank you! What's your opinion on #386?

File: fieldservice_account/models/fsm_order.py [hunk 1]
```diff
@@ -32,31 +33,45 @@ class FSMOrder(models.Model):
...
                                   domain=[('customer', '=', True)],
                                   change_default=True,
                                   index=True,
                                   track_visibility='always')
+    cost_method = fields.Selection([('timesheet', 'Timesheets'),
+                                    ('fixed', 'Fixed Rate')],
+                                   string='Cost Method',
+                                   required=True,
+                                   default='timesheet')
+    fixed_cost = fields.Float(string='Fixed Cost')
```
Comment (max3903) [inline fieldservice_account/models/fsm_order.py]: @brian10048 Why not use the product below to get the costing method from the product category and the standard cost from the product?

Comment (dreispt) [reply fieldservice_account/models/fsm_order.py]: ... and apply Price Lists to get the amount to bill.

**Review by dreispt — COMMENTED** [review]
LGTM, agree with Max on using Product data instead of a fixed price float.

File: fieldservice_account/models/fsm_order.py (hunk 1 above)
Comment (brian10048) [reply fieldservice_account/models/fsm_order.py]: @max3903 @dreispt - That makes more sense

Your suggestions have been made.

**Comment by brian10048** [comment]
Not relevant since #411

**Pull request closed without merging**